pennylane/templates/quantum_quantities/swap_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""
+Contains the SwapTest template.
+"""
+
+import pennylane as qml
+from pennylane.operation import Operation, AnyWires, Wires
+from pennylane.ops import Hadamard, CSWAP
+from pennylane.measure import sample
+
+
+class SwapTest(Operation):
+    """A class that implements the SWAPTest"""
+    num_wires = AnyWires
+    grad_recipe = None
+
+    def __init__(self, ancilla, q_reg1, q_reg2, do_queue=True, id=None):
```

Comment:
Given the history of the word "ancilla", it might be better if we use "aux" or "auxiliary" instead.

Reply:
Cool I can change it! What exactly is this 'history'?

Reply:
Roman term for a specific type of female slave. https://en.wikipedia.org/wiki/Ancillae